use_cases/eluc/data/constants.py

```diff
@@ -0,0 +1,85 @@
+from pathlib import Path
+
+import pandas as pd
+import regionmask
+
+# This should work if python path is set to ELUC
+ROOT_DIR = Path(".")
+DATA_FILE_PATH = ROOT_DIR / "data/merged_aggregated_dataset_1850_2022.zarr.zip"
+UPDATE_FILE_PATH = ROOT_DIR / "data/BLUE_LUH2-GCB2022_ELUC-committed_gridded_net_1850-2021.nc"
+CODES_PATH = ROOT_DIR / "data/codes.csv"
+
+LAND_USE_COLS = ['c3ann', 'c3nfx', 'c3per','c4ann', 'c4per', 'pastr', 'primf', 'primn', 'range', 'secdf', 'secdn', 'urban']
+
+# TODO: Temporary to merge the crops together
+CROP_COLS = ['c3ann', 'c3nfx', 'c3per','c4ann', 'c4per']
+LAND_USE_COLS = ["crop"] + [col for col in LAND_USE_COLS if col not in CROP_COLS]
+
+DIFF_LAND_USE_COLS = [f"{col}_diff" for col in LAND_USE_COLS]
+COLS_MAP = dict(zip(LAND_USE_COLS, DIFF_LAND_USE_COLS))
+
+# RECO_COLS = ['c3ann', 'c3nfx', 'c3per','c4ann', 'pastr', 'range', 'secdf', 'secdn']
+NO_CHANGE_COLS = ["c4per_diff", "primf_diff", "primn_diff", "urban_diff"]
+RECO_COLS = [col.split("_")[0] for col in DIFF_LAND_USE_COLS if col not in NO_CHANGE_COLS]
+DIFF_RECO_COLS = [f"{col}_diff" for col in RECO_COLS]
+RECO_MAP = dict(zip(RECO_COLS, DIFF_RECO_COLS))
+
+NONLAND_FEATURES = ["cell_area", "lat", "lon", "time"]
+
+# TODO: Updated this to include lat lon time
+# NN_FEATS = LAND_USE_COLS + ["cell_area"] + DIFF_LAND_USE_COLS
+NN_FEATS = LAND_USE_COLS + NONLAND_FEATURES + DIFF_LAND_USE_COLS
+
+EU_COUNTRIES = ["GB", "FR", "DE", "NL", "BE", "CH", "IE"]
+# ["Brazil", "Bolivia", "Paraguay", "Peru", "Ecuador", "Colombia", "Venezuela", "Guyana", "Suriname", "Uruguay", "Argentina", "Chile"]
+SA_COUNTRIES = ["BR", "BO", "PY", "PE", "EC", "CO", "VE", "GY", "SR", "UY", "AR", "CL"]
+US_COUNTRIES = ["US"]
+COUNTRY_DICT = {"EU": EU_COUNTRIES, "SA": SA_COUNTRIES, "US": US_COUNTRIES, "ALL": None}
+
+MANUAL_MAP = {
+    "INDO": 360,
+    "DRC": 180,
+    "RUS": 643,
+    "N": 578,
+    "F": 250,
+    "J": 388,
+    "NA": 516,
+    "PAL": 275,
+    # "J": 400,
+    "IRQ": 368,
+    "IND": 356,
+    "IRN": 364,
+    "SYR": 760,
+    "ARM": 51,
+    "S": 752,
+    "A": 36,
+    "EST": 233,
+    "D": 276,
+    "L": 442,
+    "B": 56,
+    "P": 620,
+    "E": 724,
+    "IRL": 372,
+    "I": 380,
+    "SLO": 705,
+    "FIN": 246,
+    "J": 392,
+    "BiH": 70,
+    "NM": 807,
+    "KO": 383,
+    "SS": 728
+}
+
+countries = regionmask.defined_regions.natural_earth_v5_0_0.countries_110
```

Comment:
I don't think we should have a lowercase variable name in this constants files. Looks like you can combine this line with the next.